dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -402,6 +403,33 @@ public static <T> T jsonToObject(File f, Class<T> clazz){
 		}
 	}
 
+    /**
+     * Deserialize an object back from JSON (using jackson)
+     *
+     * @param f file to deserialize
+     * @param typeReference value type to deserialize
+     * @return A deserialized object
+     */
+    public static <T> T jsonToObject(File f, TypeReference<T> typeReference){
+        ObjectMapper mapper = new ObjectMapper();
+
+        BufferedInputStream input = null;
+        try {
```

Comment:
We might use the Try-With-Resources here I think

Reply:
done